### DISCUSSION TITLE
See you at the OpenJS Foundation Collaborator Summit: Dublin in October 2

### DISCUSSION THREAD

**Discussion by jviotti** [nodejs/single-executable#42] [Announcements]
Hey there! As some of you might know already, we will be hosting an in-person + remote collaboration session around SEAs at the [OpenJS Foundation Collaborator Summit](https://www.eventbrite.com/e/openjs-foundation-collaborator-summit-dublin-tickets-410575531467). 

If you are in Dublin, come over to say hi! If not, we will welcome remote attendants too! 

Do reach out to @jviotti , @RaisinTen or @ovflowd if you have any questions!

See https://github.com/nodejs/single-executable/issues/32 for details.

**Comment by mhdawson** [comment]
Looking forward to it.
